From 33ed1f7e927bd14f8b333f73fcd6aadc7752a1da Mon Sep 17 00:00:00 2001
From: Zhang Jian <zhangjian210@huawei.com>
Date: Tue, 29 Nov 2022 13:57:30 +0000
Subject: [PATCH] cpu-feature: Enable Taisan IDC feature for Taishan core
 version
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 33ed1f7e927bd14f8b333f73fcd6aadc7752a1da
Modified-by-SEL: No


hulk inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I63SDZ

-------------------------------

Some Taishan core's sub version is 2, and add new version for Taishan core
adaptation.

Signed-off-by: Zhang Jian <zhangjian210@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/kernel/cpu_errata.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/kernel/cpu_errata.c b/arch/arm64/kernel/cpu_errata.c
index ba26ef1739a4..a4f258c83a1f 100644
--- a/arch/arm64/kernel/cpu_errata.c
+++ b/arch/arm64/kernel/cpu_errata.c
@@ -68,6 +68,7 @@ hisilicon_1980005_match(const struct arm64_cpu_capabilities *entry,
 	static const struct midr_range idc_support_list[] = {
 		MIDR_ALL_VERSIONS(MIDR_HISI_TSV110),
 		MIDR_REV(MIDR_HISI_TSV200, 1, 0),
+		MIDR_REV(MIDR_HISI_TSV200, 1, 2),
 		{ /* sentinel */ }
 	};
 
-- 
2.33.0

